translations/en/LC_MESSAGES/messages.po

```diff
@@ -20,89 +20,89 @@ msgstr ""
 
 #: translations/added_words.list
 msgid "categories"
-msgstr "Categories"
+msgstr "categories"
 
 #: translations/added_words.list
 msgid "types"
-msgstr ""
+msgstr "clothes, shoes, bedding..."
```

Comment:
Don't get this and further similar 'translations'

Reply:
In short, it's much more readable for a casual user to see types as an example of things to choose from, than just a word "types" to decipher.

Because, what is type? It's a very ambiguous word. When a user sees word type, without expanding this item they won't know if type is for male / female / kids or new / worn or natural material / synthetic material 🤷🏾 .

Reply:
Then it would mean that we do not need categories at all.
This needs further discussion. I'd split it into two commits - one which translates things as they are (which would be improvement) and second - with changed (or not) solution about categories.